### PR TITLE
add brawl as a server

### PR DESCRIPTION
**Pull request by TheRealRyGuy**
## Type of change

- [x] Add new directory for a server
- [x] Update directory of a server
- [x] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

## Further comments

Fully implement brawl.com as a server

### CODE REVIEW

File: minecraft_servers/brawl/manifest.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "server_wildcards": [
+    "%.brawl.com"
+  ],
+  "social": {
+    "web": "https://www.brawl.com/",
+    "web_shop": "https://shop.brawl.com/",
+    "web_support": "https://www.brawl.com/support/",
+    "twitter": "BrawlGames",
+    "facebook": "BrawlGames",
+	"tiktok":  "brawlwastaken",
```
Comment (scrummer) [inline minecraft_servers/brawl/manifest.json:14]: Please fix the indentation here

File: minecraft_servers/brawl/manifest.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  "server_name": "brawl",
+  "nice_name": "Brawl Games",
+  "direct_ip": "brawl.com",
+  "server_wildcards": [
+    "%.brawl.com"
+  ],
+  "social": {
+    "web": "https://www.brawl.com/",
+    "web_shop": "https://shop.brawl.com/",
+    "web_support": "https://www.brawl.com/support/",
```
Comment (scrummer) [inline minecraft_servers/brawl/manifest.json:11]: Please remove the trailing slashes for the URLs

File: minecraft_servers/brawl/manifest.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    "twitter": "BrawlGames",
+    "facebook": "BrawlGames",
+	"tiktok":  "brawlwastaken",
+    "instagram": "minecraft_brawl",
+    "youtube": "https://www.youtube.com/channel/UCDHRIcT3eQ9HEFlqGtv_9kQ",
+    "discord": "https://discord.brawl.com"
```
Comment (scrummer) [inline minecraft_servers/brawl/manifest.json:17]: Please have a look at [Manifest.md#discord-url](https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url):
Please change this URL to an official discord invite link (e.g. https://discord.gg/xyz )

**Review by scrummer — CHANGES_REQUESTED** [review]
Hi @TheRealRyGuy 

Some general changes I would like you to apply:
* The images are in general way too large. Please compress them properly (e.g. use https://compresspng.com/)
* For all the gamemode-objects: Remove trailing slashes in the URLs
* General formatting: It seems like you have quite a mixture of tabs and spaces in this file. Make sure to use always the same (no mix; we prefer spaces for JSON). And keep indentation always the same (either 2 or 4 spaces!)

Thank you, Best
Scrummer

**Comment by TheRealRyGuy** [comment]
All fixed and merged my fork! Apologies for the strange formatting, it looks completely normal in Notepad++ for some reason. Should now be fully in line with your guidelines

**Comment by scrummer** [comment]
Looks good, thank you very much!

BR
Scrummer

**Pull request merged**